src/models.js

```diff
@@ -386,7 +386,7 @@ module.exports = ({ cooler, isPublic }) => {
     return conditions.every((x) => x);
   };
 
-  const isLooseReply = (message) => {
+  const isLooseSubtopic = (message) => {
```

Comment:
Could be useful to add comments explaining what "loose" messages are / under what circumstances they occur.  I see later in this file around line 1230 these functions are used to tell apart "thread replies" from "message replies" but I'm not sure what the difference is